tutorial/build-with-logto/_connector-facebook.mdx

```diff
@@ -0,0 +1,53 @@
+### Register a Facebook developer account
+
+[Register as a Facebook Developer](https://developers.facebook.com/docs/development/register/) if you don't already have one.
+
+### Set up a Facebook app
+
+1. Visit the [Apps](https://developers.facebook.com/apps) page.
+2. Click your existing app or [create a new one](https://developers.facebook.com/docs/development/create-an-app) if needed.
+   - The selected [app type](https://developers.facebook.com/docs/development/create-an-app/app-dashboard/app-types) is up to you, but it should have the product _Facebook Login_.
+3. On the app dashboard page, scroll to the "Add a product" section and click the "Set up" button on the "Facebook Login" card.
+4. Skip the Facebook Login Quickstart page, and click the sidebar -> "Products" -> "Facebook Login" -> "Settings".
+5. In the Facebook Login Settings page, fill `${your_logto_origin}/callback/${connector_id}` in the "Valid OAuth Redirect URIs" field. The `connector_id` can be found on the top bar of the Logto Admin Console connector details page. E.g.:
+   - `https://logto.dev/callback/${connector_id}` for production
+   - `https://localhost:3001/callback/${connector_id}` for testing in the local environment
+6. Click the "Save changes" button at the bottom right corner.
+7. In the Facebook app dashboard page, click the sidebar -> "Settings" -> "Basic" and then you will get the **App ID** and **App secret**. The _App ID_ is the `clientId` in your Logto connector conifg and the _App secret_ is the `clientSecret`.
+
+### Test sign-in with Facebook's test users
+
+You can use the accounts of the test, developer, and admin users to test sign-in with the related app under both development and live [app modes](https://developers.facebook.com/docs/development/build-and-test/app-modes).
+
+You can also set the app to "live mode" by [switching modes](https://developers.facebook.com/docs/development/build-and-test/app-modes#switching-modes) so that any Facebook user can sign in with the app.
+
+- In the app dashboard page, click the sidebar -> "Roles" -> "Test Users".
+- Click the "Create test users" button to create a testing user.
+- Click the "Options" button of the existing test user, and you will see more operations, e.g., "Change name and password".
+
+### Publish Facebook sign-in settings
+
+Usually, only the test, admin, and developer users can sign in with the related app under [development mode](https://developers.facebook.com/docs/development/build-and-test/app-modes#development-mode).
+
+To enable normal Facebook users sign-in with the app in the production environment, you maybe need to switch your Facebook app to _[live mode](https://developers.facebook.com/docs/development/build-and-test/app-modes#live-mode)_, depending on the app type.
```

Comment:
```suggestion
To enable normal Facebook users sign-in with the app in the production environment, you may need to switch your Facebook app to _[live mode](https://developers.facebook.com/docs/development/build-and-test/app-modes#live-mode)_, depending on the app type.
```

Reply:
Also what is `depending on the app type`? Does it mean if the app type is something else, then the user may not have to switch the facebook app to live mode? Why?